release/models/vlan/openconfig-vlan.yang

```diff
@@ -106,6 +106,36 @@ module openconfig-vlan {
       description "Admin state of the VLAN";
     }
 
+    // Private VLAN
+    leaf private-vlan-type {
+      type enumeration {
+        enum PRIMARY {
+          description "Configure the VLAN as a primary private VLAN";
+        }
+        enum ISOLATED {
+          description "Configure the VLAN as an isolated private VLAN";
+        }
+        enum COMMUNITY {
+          description "Configure the VLAN as a community private VLAN";
+        }
+      }
+      description "Private VLAN type";
+   }
+
+  }
+
+  // Associate private VLAN to secondary VLAN
+  grouping private-vlan-association {
+    description "Mapping primary to secondary vlan ";
+
+    leaf-list community-vlan-ids {
```

Comment:
We should be prescriptive in terms of what a vendor should do in case vlans that are not community/isolated are specified incorrectly here for `community-vlan-ids` or `isolated-vlan-ids`. I am assuming we should expect the vendor to respond back with a grpc error. Let's enumerate that here in the description.

Reply:
> We should be prescriptive in terms of what a vendor should do in case vlans that are not community/isolated are specified incorrectly here for `community-vlan-ids` or `isolated-vlan-ids`. I am assuming we should expect the vendor to respond back with a grpc error. Let's enumerate that here in the description.

Will update the description.

Reply:
One more thing, can you move your comments to ->  https://github.com/openconfig/public/pull/406
That is the latest one with CLA:Yes. This pull request eems to be the older one having the CLA issues. 

Should i close this one?